doc/source/tutorials/local/Adding_new_data/Select_write_store.rst

```diff
@@ -0,0 +1,286 @@
+the data filename and the precisions filename. *Note: A simpler method
+of uploading these file types is planned.*
+
+We can now retrieve the example data for Capegrim as we did above
+
+.. code:: ipython3
+
+    cgo_url = "https://github.com/openghg/example_data/raw/main/timeseries/capegrim_example.tar.gz"
+
+.. code:: ipython3
+
+    capegrim_data = retrieve_example_data(url=cgo_url)
+
+.. code:: ipython3
+
+    capegrim_data
+
+We must create a ``tuple`` associated with each data file to link this
+to a precision file:
+
+.. code:: python
+
+   list_of_tuples = [(data1_filepath, precision1_filepath), (data2_filepath, precision2_filepath), ...]
+
+.. code:: ipython3
+
+    capegrim_data.sort()
+    capegrim_tuple = (capegrim_data[0], capegrim_data[1])
+
+The data being uploaded here is from the Cape Grim station in Australia,
+site code “CGO”.
+
+We can add these files to the object store in the same way as the DECC
+data by including the right keywords: - site code - ``"CGO"`` for Cape
+Grim - data type - ``"GCWERKS"`` - network - ``"AGAGE"``
+
+.. code:: ipython3
+
+    agage_results = standardise_surface(filepaths=capegrim_tuple, source_format="GCWERKS", site="CGO",
+                                  network="AGAGE", instrument="medusa")
+
+When viewing ``agage_results`` there will be a large number of
+Datasource UUIDs shown due to the large number of gases in each data
+file
+
+.. code:: ipython3
+
+    agage_results
+
+A note on Datasources
+^^^^^^^^^^^^^^^^^^^^^
+
+Datasources are objects that are stored in the object store (++add link
+to object store notes++) that hold the data and metadata associated with
+each measurement we upload to the platform.
+
+For example, if we upload a file that contains readings for three gas
+species from a single site at a specific inlet height OpenGHG will
+assign this data to three different Datasources, one for each species.
+Metadata such as the site, inlet height, species, network etc are stored
+alongside the measurements for easy searching.
+
+Datasources can also handle multiple versions of data from a single
+site, so if scales or other factors change multiple versions may be
+stored for easy future comparison.
+
+3. Searching for data
+---------------------
+
+Searching the object store
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We can search the object store by property using the
+``search_surface(...)`` function.
+
+For example we can find all sites which have measurements for carbon
+tetrafluoride (“cf4”) using the ``species`` keyword:
+
+.. code:: ipython3
+
+    from openghg.retrieve import search_surface
+
+    cfc_results = search_surface(species="cfc11")
+    cfc_results
+
+We could also look for details of all the data measured at the Billsdale
+(“BSD”) site using the ``site`` keyword:
+
+.. code:: ipython3
+
+    tac_results = search_surface(site="tac")
+    tac_results
+
+.. code:: ipython3
+
+    tac_results.results
+
+For this site you can see this contains details of each of the species
```

Comment:
And search will return details from both object stores? How will retrieve work here? Will it need a store to be passed for instance?

Reply:
File removed.

Reply:
SearchResults has been updated to be able to retrieve from any store results come from. A single SearchResults object is returned.